wordpress-mobile/WordPress-Android
# Implement "Enable post sharing" QS task

Fixes #13844

This task implements the "Enable post sharing" Quick Start task, which is part of the "Grow your audience" category.

This task has two steps: the first one is clicking the "Sharing" item on the My Site menu list and the second one is then selecting any item on the Sharing screen, which then completes the task.

https://user-images.githubusercontent.com/830056/105924523-2b32ce00-601d-11eb-862a-df39773d4be0.mp4

## To test

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen and select a site where Quick Start is active.
1. Under the "Grow your audience" category, select the "Enable post sharing" task.
1. Notice that:
   1. The screen scrolls to the "Sharing" item.
   1. A focus point appears over the item.
   1. The correct snackbar message appears.
1. Click the "Sharing" item.
1. On the Sharing screen, notice that:
   1. A focus point appears over the items.
   1. The correct snackbar message appears.
1. Without clicking anything, go back to the previous screen.
1. On the My Site screen, notice that the focus point has disappeared.
1. Scroll to the "Grow your audience" category.
1. Notice that the "Enable post sharing" task is still uncompleted. Select it again.
1. Repeat steps 4, 5 and 6.
1. On the Sharing screen, click any of the items.
1. Notice the Publicize details screens.
1. Go back twice.
1. On the My Site screen, scroll to the "Grow your audience" category.
1. Notice that the "Enable post sharing" task is now completed.

---

PR submission checklist:
